http_server/Files.js

```diff
@@ -0,0 +1,36 @@
+const fs = require("fs");
+const path = require("path");
+
+// module.exports =
+
+class Files {
```

Comment:
revert files.js to how it was originally, unless there is a reason you haven't done so yet

Reply:
done and files will return the path now